sentry-kubernetes/README.md

```diff
@@ -0,0 +1,27 @@
+# sentry-kubernetes
+
+[sentry-kubernetes](https://github.com/getsentry/sentry-kubernetes) is a utility that pushes Kubernetes events to [Sentry](https://sentry.io).
+
+# Installation:
+
+```console
+$ helm install incubator/sentry-kubernetes --name my-release --set sentry.dsn=<your-dsn>
```

Comment:
I think this needs to be changed

Reply:
Changed in 2f2f7adbd0343b618ce4c77c2802a4eda2043390